### PR TITLE
Spanish translation by Carlos Longarela

### PR DESCRIPTION
**Pull request by CarlosLongarela**
Spanish translation commented on forum https://wordpress.org/support/topic/translation-911/

### CODE REVIEW

**Comment by joshcanhelp** [comment]
@CarlosLongarela - Apologies for the delay on reviewing this. I had a number of updates in this [upcoming release](https://github.com/auth0/wp-auth0/milestone/19) and still need to get this reviewed on our end. #561 has updates to the translation file and that will be merged here in the next day or so. You'll also need to rebase this with `master` (should be easy, just toss all of the conflicts in your branch) or create a new PR against `master` with the updated mo/pot files (copy the ones you have, already, create a new branch from `master`, add these files, add missing translations, and create a new PR). 

Thank you again!

**Comment by joshcanhelp** [comment]
@CarlosLongarela - The other PR is now merged so POT and English translation are up-to-date and unlikely to change before the next minor release.

**Comment by joshcanhelp** [comment]
@CarlosLongarela - Just checking in here ... we're working through the last few changes before testing and release. Are you able to take a look at this in the next week or so? If not, we can push this to the next release but would love to get this in. 

Thank you!

**Comment by lbalmaceda** [comment]
@joshcanhelp I see there are no updated for this lately, and a lot of conflicts to fix. Are you willing to update this PR @CarlosLongarela ? Please ping me in that case. I'm happy to give it a detailed review!

**Comment by CarlosLongarela** [comment]
> @joshcanhelp I see there are no updated for this lately, and a lot of conflicts to fix. Are you willing to update this PR @CarlosLongarela ? Please ping me in that case. I'm happy to give it a detailed review!

Sorry I was out some time. What must I do?

**Comment by joshcanhelp** [comment]
@CarlosLongarela - No problem, Carlos, thanks for checking back in. 

We need the conflicts fixed before we can review and merge this. Instructions are [in a comment above](#issuecomment-427419433). Let me know if you need any guidance there, happy to help.

**Comment by CarlosLongarela** [comment]
@joshcanhelp please tell me if now is ok

File: tests/traits/users.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<?php
```
Comment (joshcanhelp) [inline tests/traits/users.php]: @CarlosLongarela - This file should not be added, was renamed to `usersHelper.php`

File: assets/css/initial-setup.css [hunk 2]
```diff
@@ -1 +1 @@
...ates/03.svg")}#wpwrap .a0-feedback .feedback-face.calification-4{background-image:url("../img/rates/04.svg")}#wpwrap .a0-feedback .feedback-face.calification-5{background-image:url("../img/rates/05.svg")}@-moz-keyframes loaderAnim{0%{-webkit-transform:rotate(0);-moz-transform:rotate(0);-ms-transform:rotate(0);transform:rotate(0)}100%{-webkit-transform:rotate(360deg);-moz-transform:rotate(360deg);-ms-transform:rotate(360deg);transform:rotate(360deg)}}@-webkit-keyframes loaderAnim{0%{-webkit-transform:rotate(0);-moz-transform:rotate(0);-ms-transform:rotate(0);transform:rotate(0)}100%{-webkit-transform:rotate(360deg);-moz-transform:rotate(360deg);-ms-transform:rotate(360deg);transform:rotate(360deg)}}@-o-keyframes loaderAnim{0%{-webkit-transform:rotate(0);-moz-transform:rotate(0);-ms-transform:rotate(0);transform:rotate(0)}100%{-webkit-transform:rotate(360deg);-moz-transform:rotate(360deg);-ms-transform:rotate(360deg);transform:rotate(360deg)}}@keyframes loaderAnim{0%{-webkit-transform:rotate(0);-moz-transform:rotate(0);-ms-transform:rotate(0);transform:rotate(0)}100%{-webkit-transform:rotate(360deg);-moz-transform:rotate(360deg);-ms-transform:rotate(360deg);transform:rotate(360deg)}}
```
Comment (joshcanhelp) [inline assets/css/initial-setup.css]: I'm a little worried about what happened here. It's minified so hard to see what the difference is.

**Comment by joshcanhelp** [comment]
@CarlosLongarela - Close but there are a couple of files that changed that should not have.

File: assets/css/initial-setup.css (hunk 2 above)
Comment (CarlosLongarela) [reply assets/css/initial-setup.css]: I accepted the >>>>>>> changes

Comment (joshcanhelp) [reply assets/css/initial-setup.css]: Those actually came from your merge and indicate a conflict. Try copying the file in from `master` and pushing:

https://github.com/auth0/wp-auth0/blob/master/assets/css/initial-setup.css

Comment (CarlosLongarela) [reply assets/css/initial-setup.css]: Sure, is now ok?

**Comment by CarlosLongarela** [comment]
I see 72 new strings in code, please let me 1 hour for translate it.

**Comment by CarlosLongarela** [comment]
All translated. Is now all ok?

**Comment by joshcanhelp** [comment]
Thanks so much @CarlosLongarela, I'll assign this for review now!

**Pull request merged**